maths/armstrong_numbers.py

```diff
@@ -1,10 +1,13 @@
 """
-An Armstrong number is equal to the sum of its own digits each raised
-to the power of the number of digits.
+An Armstrong number is equal to the sum of its own digits each raised to the
+power of the number of digits.  
+
 For example, 370 is an Armstrong number because 3*3*3 + 7*7*7 + 0*0*0 = 370.
-An Armstrong number is often called Narcissistic number.
-"""
 
+Armstrong numbers are also called Narcissistic numbers and Plurperfect numbers.
+
+OEIS entry: A005188

```

Comment:
What is OEIS??

Reply:
[Online Encyclopedia of Integer Sequences](https://oeis.org/). 
As the name says, it is an encyclopedia when you can search about many sequences, from the most famous as Fibonacci (A000045) to the most obscure ones as the [Look-And-Say Sequence](https://oeis.org/A005150).

Maybe a direct link would be more useful than a crude entry?